pgx-tests/src/framework.rs

```diff
@@ -256,7 +327,7 @@ fn install_extension() -> eyre::Result<()> {
         .arg("--pg-config")
         .arg(pg_config.path().ok_or(eyre!("No pg_config found"))?)
         .stdout(Stdio::inherit())
-        .stderr(Stdio::inherit())
+        .stderr(Stdio::piped())
```

Comment:
Setting this to `piped` means that we can capture it in the child process so that we can display it upon an error condition

Reply:
Is that really the thing that was swallowing all those messages? **Thank you**.

Reply:
I don't believe this particular thing was swallowing all of the test messages, but it was definitely swallowing any error messages as a result of the test framework installing the extension via `cargo pgx install ...`. The main reason for no errors showing up at all was the `std::process::exit(1)` call that I eliminated earlier.

This change was a result of me going through as many paths as possible, causing them to break, then seeing the result on the screen as the test runs. In this particular case, if `cargo pgx install ...` failed for any reason, it would fail silently, the test would attempt to continue then fail for "some other reason" that was not really visible to the user.